HW2/03-count.html

```diff
@@ -20,7 +20,7 @@ <h1>Exercise 03 - Count</h1>
         <input type="text" oninput="get_word(this.value)" id="word_find"/>
```

Comment:
WAVE Audit Error: for the input box and label set the labels "for" attribute to equal the same exact value as the inputs "id" attribute.